flutter/lib/benchmark/state.dart

```diff
@@ -392,20 +392,14 @@ class BenchmarkState extends ChangeNotifier {
       performanceRunInfo.loadgenInfo!;
 
       final performanceResult = performanceRunInfo.result;
+      if (!(performanceResult.accuracy1?.isInBounds() ?? true) ||
+          !(performanceResult.accuracy2?.isInBounds() ?? true)) {
```

Comment:
Can the check be moved to here right after we get the result, then we can save some duplicated lines?

https://github.com/mlcommons/mobile_app_open/blob/84ddef0b9e056ac030726afb63b142b4ec92b440/flutter/lib/benchmark/state.dart#L585-L588

Reply:
Done